File: A.Wrong_Subtraction.py
import math
inp=input() 
a=int(inp.split()[0])
b=int(inp.split()[1])
r=0
l=False
for i in range(b):
  if(l==False):
    r=a-i
  if(math.fmod(r,10)==0 and r!=0):
    r=r/10
    l=True
    continue
  if(math.fmod(r,10)!=0):
    r=r-1
print(int(r))
#https://codeforces.com/contest/977/problem/A #Div_3

File: A.Wrong_Subtraction.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
  int a,b;
  cin>>a>>b;
  for(int i=0;i<b;i++){
    if(a%10==0){
      a/=10;
    }else{
      a--;
    }
  }
  cout << a<<endl;
  return 0;
} 
//https://codeforces.com/contest/977/problem/A #Div_3
